doc/Makefile

```diff
@@ -5,7 +5,8 @@ default: html
 # You can set these variables from the command line.
 SRCDIR           := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 JULIAHOME        := $(abspath $(SRCDIR)/..)
-JULIA_EXECUTABLE := $(JULIAHOME)/usr/bin/julia
+include $(JULIAHOME)/Make.inc
```

Comment:
Probably not, but I couldn't find a drawback to doing things that way. It's also more consistent with other Makefiles. I can remove it if you like.